### DISCUSSION TITLE
Contribute and Collaborate

### DISCUSSION THREAD

**Discussion by wintercodeNKJJ** [elbwalker/walkerOS#238] [General]
Hi, I am new to **_open source contributions_** on **github** and would like to **kick off** on this project.
### Done
- [x] Fork project
- [x] Cloned project
- [x] Created a branch
- [ ] Install project
- [ ] Run project

### Needed
1. Installation instructions
2. Runing project instructions

**Comment by alexanderkirtzel** [comment]
Hey, just saw that you forked, good. I'll delete the other branch. You can use yours.

To get started run `npm i` and go for `npm run dev` to start developing.
Please make sure to write test files and/or update existing ones.

Don't hesitate to ask questions.

Might be interesting for others how to get started. We'll polish the docs and should add a "How to contribute" section.

**Reply by wintercodeNKJJ** [reply]
Okay thank you very much
